### PR TITLE
Add possibility to use radios instead of dropdown for enums + example page

### PR DESCRIPTION
**Pull request by florina-chesaru**

### CODE REVIEW

**Comment by schmunk42** [comment]
Looks nice! We'll take a closer look ASAP.

**Comment by florina-chesaru** [comment]
Thanks, @schmunk42! That would be great!
We are trying to integrated the JSON Editor into our project and figured there are a couple of things we need. There might be some more stuff from me coming your way. 😄

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@json-editor/json-editor",
+  "name": "@myporsche/pid-json-editor",
```
Comment (schmunk42) [inline package.json]: @florina-chesaru Umm, I think this one went on the wrong remote.

Comment (schmunk42) [reply package.json]: @florina-chesaru I think you can `push --force` to remove it or create a new PR.

Comment (florina-chesaru) [reply package.json]: Ooops, sorry for that :-)

Comment (schmunk42) [reply package.json]: No problem ;)

File: .npmignore [hunk 2]
```diff
@@ -1,2 +0,0 @@
-node_modules
-.idea
```
Comment (schmunk42) [inline .npmignore]: @florina-chesaru Can you please also re-add these files.

Comment (florina-chesaru) [reply .npmignore]: Done, sry for that.

**Comment by schmunk42** [comment]
Thank you!

**Pull request merged**